### DISCUSSION TITLE
SvelteKit + PocketBase for static Netlify app?

### DISCUSSION THREAD

**Discussion by dsm-72** [pocketbase/pocketbase#1877] [Q&A]
If I wanted to use PocketBase as a database for managing static content on a website e.g. what could otherwise be a mess of JSON files that get imported via SvelteKits `+page.ts` (**ts**) files, is that possible? 

Basically I want to use PocketBase as an admin panel for managing / updating static data and then via SvelteKit query (the frozen data for deployment) for different routes / filtering etc.

Does this work at all?

**Comment by ganigeorgiev** [comment]
If I'm understanding your use case correctly, as long as you can connect to the PocketBase instance during the static app build process so that it can fetch it's required data, I don't see why it shouldn't work.

The JS SDK support both auth record (aka. `pb.collection("users").authWithPassword(...)`) and admin level authentication (aka. `pb.admins.authWithPassword(...)`). For your particular use case to fetch data during the static build, authenticating as admin should be sufficient.

**Reply by dsm-72** [reply]
Hmm. Do you know if it is possible to deploy PocketBase with netlify?

**Reply by ganigeorgiev** [reply]
I haven't used Netlify and I'm not sure if it is possible.

But once again, for your use case with the static data you may not need it. You can run PocketBase locally, build your static app and deploy the generated static assets to Netlify.
